Brokerages/Bitfinex/BitfinexBrokerage.Messaging.cs

```diff
@@ -561,13 +561,10 @@ private void EmitFillOrder(string[] entries)
                 var fillQuantity = decimal.Parse(entries[5], NumberStyles.Float, CultureInfo.InvariantCulture);
                 var direction = fillQuantity < 0 ? OrderDirection.Sell : OrderDirection.Buy;
                 var updTime = Time.UnixTimeStampToDateTime(double.Parse(entries[3], NumberStyles.Float, CultureInfo.InvariantCulture));
-                var orderFee = OrderFee.Zero;
-                if (fillQuantity != 0)
-                {
-                    var security = _securityProvider.GetSecurity(order.Symbol);
-                    orderFee = security.FeeModel.GetOrderFee(
-                        new OrderFeeParameters(security, order));
-                }
+                var orderFee = new OrderFee(new CashAmount(
+                        Math.Abs(decimal.Parse(entries[9], NumberStyles.Float, CultureInfo.InvariantCulture)),
```

Comment:
Thanks @Adalyat! Getting close to having Bitfinex live. 

Can we get a fee for a currency we don't have an asset for? Let's check what this is before passing back. What happens when the cashbook get an unknown currency?

Reply:
Hello!
Currenty this situation is handled by EnsureCurrencyDataFeeds method and throw exception on init.
tested for this:
Account currency: ETH
Trading: BTCUSD

![2019-02-05_00-14-18](https://user-images.githubusercontent.com/3648114/52237766-78bb4b00-28db-11e9-92fb-6bcb82ecd445.png)
 
Is it expected behavior?

Reply:
@Adalyat It's a Lean bug (unrelated to Bitfinex brokerage), opened a new issue here: https://github.com/QuantConnect/Lean/issues/2885

Reply:
@StefanoRaggi  and @jaredbroad 
Just tested another case:

```
SetAccountCurrency("USD");
AddCrypto("ETHBTC", Resolution.Minute, Market.Bitfinex);
```
everything works fine and ETH fee currency handled properly
![2019-02-05_21-03-39](https://user-images.githubusercontent.com/3648114/52294250-d908c600-2989-11e9-82bb-f5b263b1d89d.png)